bin/p-s.js

```diff
@@ -55,7 +56,23 @@ runPackageScript({
 function getPSConfig() {
   var psConfigFilename = program.config ? resolve(process.cwd(), program.config) : findUp.sync('package-scripts.js')
   try {
-    return require(psConfigFilename)
+    // Require a module like babel-register
+    if (program.require) {
+      try {
+        require(program.require)
+      } catch (e) {
+        log.warn({
+          message: colors.yellow('Unable to preload ' + program.require),
+          ref: 'preload-module'
```

Comment:
This `ref` should reference a line in [ERRORS_AND_WARNINGS.md](https://github.com/kentcdodds/p-s/blob/master/other/ERRORS_AND_WARNINGS.md). Similar to how [this](https://github.com/kentcdodds/p-s/blob/9ae2217d365c06153d47db2acfd63230fcff83fb/bin/p-s.js#L62) references [this](https://github.com/kentcdodds/p-s/blob/9ae2217d365c06153d47db2acfd63230fcff83fb/other/ERRORS_AND_WARNINGS.md#unable-to-find-config). If you could add a line to ERRORS_AND_WARNINGS.md, that'd be great 👍